schema.json

```diff
@@ -1108,70 +1148,113 @@
     "name": "Products",
     "settings": [
       {
-        "type": "checkbox",
-        "label": "Show Quickview",
-        "force_reload": true,
-        "id": "show_product_quick_view"
+        "type": "heading",
+        "content": "Product sale badges"
       },
       {
-        "type": "heading",
-        "content": "Product pages"
+        "type": "select",
+        "label": "Show product sale badges",
+        "id": "product_sale_badges",
+        "force_reload": true,
+        "options": [
+          {
+            "value": "none",
+            "label": "None"
+          },
+          {
+            "value": "topleft",
+            "label": "Top Left"
+          },
+          {
+            "value": "sash",
+            "label": "Diagonal"
+          },
+          {
+            "value": "burst",
+            "label": "Burst"
+          }
+        ]
       },
       {
         "type": "text",
-        "label": "Product sale price label",
-        "id": "pdp-sale-price-label"
+        "label": "Product price label (sale)",
+        "force_reload": true,
+        "id": "pdp-sale-price-label",
+        "reference": "product_sale_badges",
+        "reference_default": "none"
       },
       {
         "type": "text",
-        "label": "Product before sale price label",
-        "id": "pdp-non-sale-price-label"
+        "label": "Product price label (before sale)",
+        "force_reload": true,
+        "id": "pdp-non-sale-price-label",
+        "reference": "product_sale_badges",
+        "reference_default": "none"
       },
       {
         "type": "text",
-        "label": "Product retail price label",
-        "id": "pdp-retail-price-label"
+        "label": "Product price label (retail)",
+        "force_reload": true,
+        "id": "pdp-retail-price-label",
+        "reference": "product_sale_badges",
+        "reference_default": "none"
       },
       {
         "type": "text",
         "label": "Product price label",
-        "id": "pdp-price-label"
+        "force_reload": true,
+        "id": "pdp-price-label",
+        "reference": "product_sale_badges",
+        "reference_default": "none"
       },
       {
         "type": "paragraph",
-        "content": "* the 'Product price label' is displayed when there is not a sale price."
+        "content": "* the 'Product price label' is displayed when there is not a sale price.",
```

Comment:
Won't we need force reload for type `paragraph` ?

Reply:
I believe this is just a <p> element, and shouldn't need a force reload as we are not making any changes. @pedelman